components/post_profile_picture/post_profile_picture.tsx

```diff
@@ -112,12 +114,11 @@ export default class PostProfilePicture extends React.PureComponent {
                 profileSrc={profileSrc}
                 isEmoji={isEmoji}
                 status={status}
-                userId={user ? user.id : null}
+                userId={user ? user.id : undefined}
                 channelId={post.channel_id}
-                username={user ? user.username : null}
+                username={user ? user.username : undefined}
```

Comment:
Could we use `user.?username` here instead?

Reply:
Hi, I already change it to  `user?.username`. Thank you.